packages/ui-components/src/components/single-select-dropdown/single-select-dropdown.helper.ts

```diff
@@ -1,11 +1,11 @@
-import { isEmpty } from 'lodash';
+import { isBoolean, isEmpty } from 'lodash';
 import { ISelectMultiOption, ISelectSingleOptions } from '../../types';
 
 export const buildSingleSelectOptions = (options: ISelectSingleOptions = {}): Record<string, ISelectMultiOption> =>
 	Object.values(options).reduce<Record<string, ISelectMultiOption>>((accumulator, option) => {
 		accumulator[option.value] = {
 			...option,
-			selectable: isEmpty(option.options),
+			selectable: isBoolean(option.selectable) ? option.selectable : isEmpty(option.options),
```

Comment:
@djcouto I introduced the selectable logic here. If you define the option as selectable or not, we will respect the prop. The fallback is the same as it was before (when we dont define the selectable prop).